### ISSUE TITLE
Align Create a Wallet Account examples

### ISSUE THREAD

**Issue by jlvandenhout** [iotaledger/iota-sdk#125]
## Description

- [ ] Either use `.env` or clearly set a required constants near the top of the example.
- [ ] Only create one account in Node.js.
- [ ] Use same paths for Stronghold and database locations across languages.
- [ ] Print similar expected output across languages.

## Are you planning to do it yourself in a pull request?

No.

**Comment by Alex6323** [comment]
I am currently waiting on more info on this from @jlvandenhout . As far as I can tell we use `.env` in almost all of our examples, and the other items on the list aren't really actionable atm as long as not all bindings (#307 in particular) have been merged into the sdk.

**Comment by jlvandenhout** [comment]
> Either use .env or clearly set a required constants near the top of the example.

Regarding this, what I specifically mean is for example in the Python example it is really hard to tell what is supposed to be replaced by the user and why, for someone who isn't familiar with the libraries. I'd at least have the password and seed as clearly named and commented variables near the top of the example, so it is immediately clear to the user what they are and why they can or should be changed. IMO the same goes for the network URL, storage paths, and the account alias. See [the getting started](https://github.com/iotaledger/iota-sdk/blob/develop/bindings/python/examples/wallet/getting-started.py) what that would look like. The same goes for NodeJS.
